src/core/geocms/geonode/qgsgeonoderequest.h

```diff
@@ -86,6 +88,8 @@ class CORE_EXPORT QgsGeoNodeRequest : public QObject
       QString wfsURL;
       //! XYZ tileserver URL for layer
       QString xyzURL;
+      //! Backend server (geoserver or qgis-server)
+      QString server;
```

Comment:
Should this be an enum instead?

Reply:
yeah I agree I think that is a better type. thanks!